docs/docs/reference/cloud/image-cdn.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Image CDN
```

Comment:
I don't see the value of this document at all and I think it shouldn't be added.

- The benefits are described in the how-to guide
- The short explanation on how it works can be added to the how-to guide
- Plan Limits will be outdated shortly and can just live on the pricing page and be linked to
- "Limitations" are mentioned in the how-to guide

Reply:
I tend to agree here on all points. Moving the content accordingly and we'll retire the reference guide.

Reply:
Still needs to be done